roles/common/defaults/main.yml

```diff
@@ -43,7 +46,6 @@ common:
   logging:
     debug: False
     verbose: True
-  serial_console: ttyS0
```

Comment:
Well now you've changed the format of the `serial_console` variable, but haven't accounted for that in the places that make use of it, which is why the PR failed.